Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = [
 ]
 
 [workspace.package]
-authors = [ "Steve Springett <steve.springett@owasp.org>" ]
+authors = [ "Steve Springett <steve.springett@owasp.org>", "Amy Keibler <3483663+amy-keibler@users.noreply.github.com>", "Sergey \"Shnatsel\" Davidoff <shnatsel@gmail.com>"]
```

Comment:
Not sure if we want to put Amy in here without her consent.
I don't have any strong opinions. @amy-keibler ping, just in case